src/content/docs/apm/agents/php-agent/configuration/distributed-tracing-php-agent.mdx

```diff
@@ -125,17 +120,21 @@ Standard distributed tracing for APM agents ([above](#quick-start-apm)) captures
 To learn more about this feature, see [Infinite Tracing](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing).
```

Comment:
The same as the others. I'd remove this tip and add the link directly to Infinite Tracing.

Reply:
@nbaenam I fixed this!